web-app-router/scenes/Portal/Profile/Teams/page/TransferTeamDialog/index.tsx

```diff
@@ -0,0 +1,47 @@
+import { Dialog, DialogProps } from "@/components/Dialog";
+import { DialogPanel } from "@/components/DialogPanel";
+import { DialogOverlay } from "@/components/DialogOverlay";
+import { CircleIconContainer } from "@/components/CircleIconContainer";
+import { ExchangeIcon } from "@/components/Icons/ExchangeIcon";
+import { DecoratedButton } from "@/components/DecoratedButton";
+import { TYPOGRAPHY, Typography } from "@/components/Typography";
+
+type TransferTeamDialogProps = DialogProps & {};
+
+export const TransferTeamDialog = (props: TransferTeamDialogProps) => {
+  const { ...otherProps } = props;
+  return (
+    <Dialog {...otherProps}>
+      <DialogOverlay />
+
+      <DialogPanel className="w-[36.25rem] grid gap-y-8">
+        <CircleIconContainer variant="info">
+          <ExchangeIcon />
+        </CircleIconContainer>
+
+        <Typography as="h3" variant={TYPOGRAPHY.H6}>
+          Transfer ownership
+        </Typography>
+
+        <p className="leading-6 font-500 text-16 text-center text-grey-500">
```

Comment:
@michaelfurutin Transfer modal is not finished, it requires a form and copy update.